core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -940,6 +946,15 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean) extends Abstra
     if (millis < 0) return -1
     millis
   }
+  
+  private def getMemoryMappedFileUpdatesEnabled: Boolean = {
+    try {
+      getBoolean(KafkaConfig.MemoryMappedFileUpdatesEnabledProp)
+    }
+    catch {
+      case e: Exception => !Os.isWindows
```

Comment:
Can we use Defaults.MemoryMappedFileUpdatesEnabled instead of !Os.isWindows here?